packages/components/pages/f-checkout/src/components/_tests/Accordion.test.js

```diff
@@ -0,0 +1,88 @@
+import Vuex from 'vuex';
+import { VueI18n } from '@justeat/f-globalisation';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Accordion from '../Accordion.vue';
+import { i18n, createStore } from './helpers/setup';
+
+const localVue = createLocalVue();
+
+localVue.use(VueI18n);
+localVue.use(Vuex);
+
+describe('Accordion', () => {
+    const propsData = { title: 'Accordion Header', id: 'note' };
+
+    it('should be defined', () => {
+        // Arrange & Act
+        const wrapper = shallowMount(Accordion, {
+            i18n,
+            store: createStore(),
+            localVue,
+            propsData
+        });
+
+        // Assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    describe('methods ::', () => {
+        describe('toggle ::', () => {
+            let wrapper;
+
+            it.each([
+                [true, false],
+                [false, true]
+            ])('should change `isExpanded` to %s when it is initially set to %s', (expected, initial) => {
+                // Arrange
+                wrapper = shallowMount(Accordion, {
+                    i18n,
+                    store: createStore(),
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            isExpanded: initial
+                        };
+                    }
+                });
+
+                // Act
+                wrapper.vm.toggle();
+
+                // Assert
+                expect(wrapper.vm.$data.isExpanded).toBe(expected);
+            });
+        });
+
+        describe('template ::', () => {
+            let wrapper;
+
+            it.each([
+                [true, false],
+                [false, true]
+            ])('should toggle the accordion visibility from %s to %s when heading is clicked', (expected, initial) => {
```

Comment:
Nit: The order of the parameters doesn't match how they're ordered in the test description

```suggestion
            ])('should toggle the accordion visibility from %s to %s when heading is clicked', (initial, expected) => {
```